docs/docs/index.md

```diff
@@ -1,12 +1,20 @@
 # Welcome
 
-Hydra is a layer-two scalability solution for Cardano, which aims to increase
-the speed of transactions (low latency, high throughput) and minimize
-transaction cost.
+This section is a manual for users interested in running a Hydra node.
 
-This is the user manual for `hydra-node`, an implementation of the Hydra Head protocol, as originally published [here](https://eprint.iacr.org/2020/299.pdf). The `hydra-node` interfaces the Cardano blockchain, connects to other `hydra-node`s on a dedicated overlay network, runs a simplified (coordinated) Hydra Head protocol, and provides an API to clients.
+Hydra node is an implementation of the Hydra Head protocol, designed to enable fast and low-cost transactions by operating off-chain while leveraging the security of the main Cardano blockchain.
 
-:::warning Mainnet Availability disclaimer
+Decentralized applications (DApps), exchanges, and enterprise-level services can choose to run a Hydra node to participate in forming Hydra Heads – mini-ledgers that process transactions independently and in parallel, significantly enhancing throughput and reducing latency. 
+
+To run a Hydra node, participants should:
+
+* **Install and configure the node**: set up the Hydra node software, ensuring it is properly configured to connect with other Hydra nodes and the Cardano mainnet.
+* **Participate in Hydra heads**: collaborate with other nodes to create and manage Hydra heads, enabling batch processing of transactions off-chain.
+* **Synchronize with Cardano**: maintain synchronization with the main chain to ensure the security and finality of transactions processed by the Hydra heads.
```

Comment:
These three bullets are weird / wrong.

1. To run a hydra node, you need to run the hydra node software .. of course?
2. Collaborate is off here IMO. It implies that it's an interactive process, but it can be also very much automated. More importantly, once set up the `hydra-node` serves as an agent as it validates transactions all by itself.
3. Finality does not depend on mainchain access. It also sounds like there is no Cardano connection needed before this step.

Do we need this paragraph? What does it even explain?

Reply:
I'll delete these bullet points and we'll discuss it with Franco as you're OOO.